client/extensions/woocommerce/app/dashboard/pre-setup-view.js

```diff
@@ -75,16 +130,31 @@ class PreSetupView extends Component {
 	}
 }
 
-function mapStateToProps( /* state */ ) {
+function mapStateToProps( state ) {
+	const site = getSelectedSiteWithFallback( state );
+	let name = '';
+	if ( site ) {
+		name = getSiteTitle( state, site.ID );
```

Comment:
I'd like to keep it this way until I get more comfortable with using the && and || operator this way. I've seen some behavior lately that I don't completely understand.